handlers/admin/adminServerStatsPage.go

```diff
@@ -125,5 +125,5 @@ func (h *Handlers) AdminServerStatsPage(w http.ResponseWriter, r *http.Request)
 	}
 	data.Registries.UploadProviders = upload.ProviderNames()
 
-	handlers.TemplateHandler(w, r, "admin/serverStatsPage.gohtml", data)
+	handlers.TemplateHandler(w, r, "serverStatsPage.gohtml", data)
```

Comment:
@jules, this page must be in `admin/` as it's admin move the page rather than change this if required.